tests/test_sort_array.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import numpy as np
+from src.mds_array_manipulation.mds_array_manipulation import sort_array
+
+# The following test is for the function sort_array
+
+# checks if a ValueError is raised for non-1D arrays
+def test_input_not_1d_array_1(two_d_int_array):
+    with pytest.raises(ValueError):
+        sort_array(two_d_int_array)
+
+def test_input_not_1d_array_2(three_d_int_array):
+    with pytest.raises(ValueError):
+        sort_array(three_d_int_array)
+
+# checks if the function correctly handles inputs that are not numpy arrays
+def test_input_not_numpy_array():
+    non_numpy_inputs = [
```

Comment:
Should we put non numpy inputs in conftest.py so we will use the same example?

Reply:
Yes, we can add it